### DISCUSSION TITLE
知识管理PKM/管理大量笔记的最佳实践？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#31] [实用技巧/最佳实践]
挖坑，我看到的概念有 PARA / LYT / zettelkasten，大家来分享一下你的方法呀！

**Comment by makerjackie** [comment]
自己推荐一个：
MOC: Map Of Content **笔记地图**（来源于 [LYT-linking your thinking笔记方法](https://forum.obsidian.md/t/lyt-kit-now-downloadable/390)）

我的理解：
* MOC跟 TOC (Table Of Content) 大纲/目录 不同的是
  * 内容自由组织，无需像目录那样按层级结构来
  * 你可以使用多个笔记地图来链接到同个笔记（相当于不同角度看笔记）
* MOC的优点
  * 组织笔记链接变成笔记地图 的过程，其实也是思维碰撞，激发灵感的过程
  * 笔记地图可以作为一个很好的（电子笔记/数字图书馆）的索引，今后（哪怕是几年后），你也能快速通过不同的MOC找到你的笔记

总而言之，用起来的方法就是创建一个像下图一样的（Home主页-笔记地图）
![image](https://user-images.githubusercontent.com/18050469/102607307-0b2d0680-4163-11eb-92a3-26269630fc4e.png)

**Reply by lillian-who** [reply]
看到PRPA立刻使用搜索打开了我关于PRPA的笔记。当时看过以后没有特别好的idea。我在obsidian的使用中，没有根据项目去分区，只分了工作和个人。个人里面日记一个文件夹，知识管理一个文件夹。知识管理文件夹是按照领域分区的，比如“如何阅读”、“爱”、“效率”这种。

我不会把资源和档案放进来。

现在摸索到最舒服的方式，其实是记日记，哈哈哈。不是常规意义上的日记，用obsidian记流水账特别合适，任何想到的想法，一个知识点，一个idea，一个待办，一个小确幸，快速的在obsidian中记下来。然后，对于某个知识点，如果篇幅大的话要展开，就新建一个页面，用![[title]]引进来。

我觉得块引用和页面引用简直绝了。

以上，我个人的观点是：系统可以尽量简化，用着最顺手的，就是对自己最有用的。

也**感谢楼主的分享**，想要尝试一下你的方式了。只是我没有很多的电子文档，这么多年的笔记都留在了各个笔记本里。

**Comment by Quorafind** [comment]
或者用 P.A.R.A 的方式进行管理

**Reply by Quorafind** [reply]
各位有想法的可以到专门的帖子下回复：https://github.com/obsidianzh/forum/discussions/39

**Comment by dearxs** [comment]
等我先去了解了解 LYT 连接你的思想 的那套东西。
我现在是不分类，我的笔记质量和数量还没有达到需要各种上层框架的程度。
zettelkasten 好像也不建议自上而下分类，而是让笔记自然聚合成主题（如果聚合不起来，那肯定是你笔记做的不够）。

**Comment by kingsun1980** [comment]
zettelkasten鼓励自下而上的生长，这个和LYT里的MOC其实是一个意思，MOC实际上是一个桥梁，把原子笔记和一定规模后的认知涌现之间搭起一个桥梁，值得仔实际中使用
